WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsViewController.swift

```diff
@@ -32,24 +32,14 @@ extension NSNotification.Name {
         guard let post = post else {
             return .init()
         }
-
-        if FeatureFlag.commentModerationUpdate.enabled {
-            let headerView = CommentTableHeaderView(title: post.titleForDisplay(),
-                                                    subtitle: .commentThread,
-                                                    showsDisclosureIndicator: true)
-            headerView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleHeaderTapped)))
-            return headerView
+        let headerView = CommentTableHeaderView(
+            title: post.titleForDisplay(),
+            subtitle: .commentThread,
+            showsDisclosureIndicator: allowsPushingPostDetails
+        ) { [weak self] in
+            self?.handleHeaderTapped()
         }
-
-        let cell = CommentHeaderTableViewCell()
-        cell.backgroundColor = .systemBackground
-        cell.configure(for: .thread, subtitle: post.titleForDisplay(), showsDisclosureIndicator: allowsPushingPostDetails)
-        cell.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleHeaderTapped)))
-
-        // the table view does not render separators for the section header views, so we need to create one.
-        cell.contentView.addBottomBorder(withColor: .separator, leadingMargin: tableView.separatorInset.left)
-
-        return cell
+        return headerView
     }
```

Comment:
The reason behind this change is that sometimes the existing table header view disappears ( see screenshots below ). I was going to fix this issue but I found there is a new header view hidden behind a feature flag. I've tested it, and it works well, but not sure why the feature flag never got enabled. cc @dvdchr 

| Expected | Actual |
| --------- | ------- | 
| ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/4f603f61-20a6-4217-a548-df701e14160c) | ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/65324ce3-441b-43a5-98b8-4a3e201bbaf3) |

Reply:
Oh, snap. I've mistakenly deleted my comment! I thought I double-posted... 😅  In essence: 

> The new comment header was part of #20568, which was an unplanned/unscheduled project that I took on during my downtime. The new design was supposed to be implemented on all three surfaces (site comments, comment threads, and comment notifications), but I only managed to ship for the comment threads. The header feature flag was disabled for consistency. Next week is our team's Maintenance Week, so I should be able to allocate some time to implement this for Site Comments.

Reply:
I see we've removed the feature flag check. Does this mean we no longer need it?

Reply:
Personally I don't think we need it, I'm considering this header redesign similar to the redesigns we are doing now, which are not hidden behind a feature flag. Is that okey with you @dvdchr ?

Reply:
Sorry for the late reply, but yes feel free to remove it. The feature flag was intended for the larger comments moderation update work, but we never really got the green light to work on it. Might as well remove it and will properly restart it when we can "officially" work on it again! 🙂